packages/vite-plugin-dusk/src/plugins/detectExternalImports.ts

```diff
@@ -143,7 +143,7 @@ File: ${withOptionalExtension} was imported by:\n${importPath.join("\n")}
 
       const isInWhitelist =
         DEPENDENCY_WHITELIST.includes(name) ||
-        options.ignoredDependencies?.includes(name)
```

Comment:
Hey @dmecke. Thank you for your contribution! I've assumed from the issue you've experienced, that we'd want to allow imports of multiple files from one dependency:

For instance if we allow dependency `lodash`, then importing `lodash/x`, `lodash/y` is fine, but importing a dependency `lodash-different-package` should not be allowed.

Could you update the code so that we allow package + package/inner/files ?

Reply:
All good, I've updated the tests. Thank you :)